ludwig/datasets/README.md

```diff
@@ -0,0 +1 @@
+## Ludwig Datasets API
```

Comment:
Are you planning to write a README here, or in ludwig-docs?

Reply:
Yes, I meant to add some basic instructions here and forgot to finish :-).  Will do that now.  Also created a task to update "Add a Dataset" in ludwig-docs.